README.md

```diff
@@ -21,10 +25,15 @@ MONAI is a [PyTorch](https://pytorch.org/)-based, [open-source](https://github.c
 - multi-GPU data parallelism support.
 
 ## Installation
-Clone and build this repository from source:
-  ```bash
-  pip install git+https://github.com/Project-MONAI/MONAI#egg=MONAI
-  ```
+To install [the current release](https://pypi.org/project/monai/):
+```bash
+pip install monai
+```
+
+To install from the source code repository:
+```bash
+pip install git+https://github.com/Project-MONAI/MONAI#egg=MONAI
```

Comment:
Is `#egg=MONAI` necessary in the URL?

Reply:
Hi @ericspod , this is not code from this PR, it's from Tom's suggestion before.
Thanks.